Editor/AGS.Editor/ImportExport.cs

```diff
@@ -694,9 +694,9 @@ public static void ExportCharacter272(Character character, string fileName, Game
             writer.Write((int)6);
             for (int i = 0; i < 256; i++)
             {
-                writer.Write((byte)(game.Palette[i].Colour.R / 4));
-                writer.Write((byte)(game.Palette[i].Colour.G / 4));
-                writer.Write((byte)(game.Palette[i].Colour.B / 4));
+                writer.Write((byte)(game.Palette[i].Colour.R));
+                writer.Write((byte)(game.Palette[i].Colour.G));
+                writer.Write((byte)(game.Palette[i].Colour.B));

```

Comment:
You have adjusted palette fixups in ExportCharacter272, but not ImportCharacter272. Is that intentional?

Personally, I believe that these two must not be adjusted, because they relate to a old data format (and by changing palette range you are changing format).
Either keep these as-is (maybe with some commentary), or remove both functions completely as obsolete.